tools/c7n_gcp/tests/test_query.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_gcp.query import GcpLocation
+from gcp_common import BaseTest
+
+
+class GcpLocationTest(BaseTest):
+    _app_locations = ["asia-east2",
```

Comment:
this test feels a little suspect in implementation, its basically a copy and paste of the data to compare, and poking at the private variables to do so.